src/genie/libs/parser/iosxe/tests/ShowIpNbarDiscovery/cli/equal/golden_output1_output.txt

```diff
@@ -0,0 +1,77 @@
+unknown                      3544                     15                      
```

Comment:
it looks like output is snipped. missing `GigabitEthernet1` and other lines.

Reply:
Yes looks like a copy/paste issue. I'll fix that one.

Reply:
confirmed the fix